packages/ui/locales/es/views.json

```diff
@@ -146,11 +146,13 @@
     "repoTagDescriptionPlaceholder": "Enter a description of this tag...",
     "creatingTagButton": "Creating tag...",
     "createTagButton": "Create tag",
-    "tag": "Tag",
-    "commit": "Commit",
     "createBranch": "Create branch",
     "viewFiles": "View Files",
     "deleteTag": "Delete tag",
+    "tag": "Tag",
+    "commit": "Commit",
+    "tagger": "Tagger",
+    "creationDate": "Creation date",
```

Comment:
I was asked to provide translations in my PR. I think you'd better do it before you open a PR in Harness's repo